common/scala/src/main/scala/whisk/core/entity/WhiskActivation.scala

```diff
@@ -90,6 +90,7 @@ case class WhiskActivation(
         }
     }
 
+    def withoutLogsOrResponse = copy(response = ActivationResponse.success(), logs = ActivationLogs()).revision[WhiskActivation](rev)
```

Comment:
Falling back to a success always breaks the health-protocol doesn't it? We can safely drop the user's response stuff, but we need to send the correct statuscode I think.